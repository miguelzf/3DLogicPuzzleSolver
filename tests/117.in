- - - - -
- R - - -
- - - - -
- - - P -
- - - L B

- - P - -
- - B - -
- - - - -
- - - - -
- - - - -

- - Y - -
R - - - -
- - - - -
- L - G -
G - Y - -
