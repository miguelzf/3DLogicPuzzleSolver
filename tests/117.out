
Solution:
g  g  g  g  g  
g  R  r  r  r  
g  b  b  b  b  
g  b  p  P  b  
g  b  p  L  B  
=============
g  b  P  l  y  ||  y  y  Y  r  g  
g  b  B  l  y  ||  R  r  r  r  g  
g  -  -  l  y  ||  y  y  y  -  g  
g  -  -  l  l  ||  l  L  y  G  g  
g  g  g  g  g  ||  G  -  Y  -  -  
